pkg/minikube/bootstrapper/bsutil/testdata/v1.15/dns.yaml

```diff
@@ -50,4 +50,5 @@ evictionHard:
 ---
 apiVersion: kubeproxy.config.k8s.io/v1alpha1
 kind: KubeProxyConfiguration
+clusterCIDR: ""
```

Comment:
Having clusterCIDR set to an empty string feels wrong here. See previous comment.

Reply:
I've based this config block on the existing ones, since clusterCIDR and podSubnet share  the same value

e.g: https://github.com/kubernetes/minikube/blob/master/pkg/minikube/bootstrapper/bsutil/testdata/v1.15/dns.yaml#L37
```yaml
...
networking:
  dnsDomain: 1.1.1.1
  podSubnet: ""
  serviceSubnet: 10.96.0.0/12
...
```

Reply:
OK, that seems reasonable. I do think podSubnet is probably doing it wrong as well though.

Reply:
Got you, i'm doing a refactor PR in the coming days to address these issues